examples/rondpoint/tests/test_generated_bindings.rs

```diff
@@ -2,8 +2,8 @@ uniffi_macros::build_foreign_language_testcases!(
     "src/rondpoint.udl",
     [
         "tests/bindings/test_rondpoint.kts",
-        "tests/bindings/test_rondpoint.swift",
-        "tests/bindings/test_rondpoint.py",
-        "tests/bindings/test_rondpoint.rb",
+        // "tests/bindings/test_rondpoint.swift",
+        // "tests/bindings/test_rondpoint.py",
+        // "tests/bindings/test_rondpoint.rb",
```

Comment:
(Just noting in case you weren't aware, it's possible to run just the kotlin tests by using `cargo test kt` without commenting out the ones for the other languages)

Reply:
This has helped immensely.